Sources/Hummingbird/Middleware/LogRequestMiddleware.swift

```diff
@@ -72,28 +72,28 @@ public struct LogRequestsMiddleware<Context: RequestContext>: RouterMiddleware {
                 level: self.logLevel,
                 "Request",
                 metadata: [
-                    "hb_uri": .stringConvertible(request.uri),
-                    "hb_method": .string(request.method.rawValue),
+                    "hb.uri": .stringConvertible(request.uri),
+                    "hb.method": .string(request.method.rawValue),
                 ]
             )
         case .all(let except):
             context.logger.log(
                 level: self.logLevel,
                 "Request",
                 metadata: [
-                    "hb_uri": .stringConvertible(request.uri),
-                    "hb_method": .string(request.method.rawValue),
-                    "hb_headers": .stringConvertible(self.allHeaders(headers: request.headers, except: except)),
+                    "hb.uri": .stringConvertible(request.uri),
+                    "hb.method": .string(request.method.rawValue),
+                    "hb.headers": .stringConvertible(self.allHeaders(headers: request.headers, except: except)),
                 ]
             )
         case .some(let filter):
             context.logger.log(
                 level: self.logLevel,
                 "Request",
                 metadata: [
-                    "hb_uri": .stringConvertible(request.uri),
-                    "hb_method": .string(request.method.rawValue),
-                    "hb_headers": .stringConvertible(self.filterHeaders(headers: request.headers, filter: filter)),
+                    "hb.uri": .stringConvertible(request.uri),
+                    "hb.method": .string(request.method.rawValue),
+                    "hb.headers": .stringConvertible(self.filterHeaders(headers: request.headers, filter: filter)),
```

Comment:
```suggestion
                    "http.request.path": .stringConvertible(request.uri),
                    "http.request.method": .string(request.method.rawValue),
                    "http.request.headers": .stringConvertible(self.filterHeaders(headers: request.headers, filter: filter)),
```

This has the same considerations as above